dace/sdfg/utils.py

```diff
@@ -1030,6 +1032,119 @@ def get_next_nonempty_states(sdfg: SDFG, state: SDFGState) -> Set[SDFGState]:
     return result
 
 
+def is_hbm_array(array: dt.Data):
```

Comment:
aren't these more FPGA-codegen-related and should be there? (e.g., `dace/codegen/targets/fpga/utils.py`)

Reply:
moved it to targets/fpga_helper/fpga_utils (fpga_helper so it does not clash with fpga.py)